src/app/machines/components/MachineForms/MachineActionFormWrapper/MachineActionFormWrapper.tsx

```diff
@@ -40,29 +41,38 @@ type Props = Omit<MachineActionFormProps, "processingCount"> & {
   viewingDetails: boolean;
 };
 
-/**
- * Displays specified machine action form for selected machines.
- */
-export const MachineActionFormWrapper = ({
+type Props = ContainerProps & {
+  clearSelectedMachines: () => void;
+  dispatch: Dispatch<AnyAction>;
```

Comment:
`dispatch` had to be added individually, since there was a type mismatch if just relying on the return type of `useSelectedMachinesActionsDispatch`